models/twitter_test.go

```diff
@@ -0,0 +1,41 @@
+package models
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTwitterUserList(t *testing.T) {
+	list := &TwitterUserList{}
+	assert.Equal(t, len(*list), 0, "should not contain any elements before create")
```

Comment:
The language guarantees that this test cannot possibly ever fail - You can just get rid of this case